backend/alembic/versions/983c29b7f0d6_add_cascade_delete_constraint_to_.py

```
@@ -0,0 +1,41 @@
+"""Add CASCADE delete constraint to messages
+
+Revision ID: 983c29b7f0d6
+Revises:
+Create Date: 2025-08-11 14:14:39.000000
+
+"""
+from alembic import op
+import sqlalchemy as sa
```

Ruff (0.12.2)

`sqlalchemy` imported but unused

Remove unused import: `sqlalchemy`

(F401)